plugins/terminal/pluma-terminal.metainfo.xml.in

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Igor Gnatenko <i.gnatenko.brain@gmail.com> -->
+<component type="addon">
+  <id>pluma-terminal</id>
+  <extends>org.pluma.mate.desktop</extends>
+  <name>Terminal</name>
+  <summary>A simple terminal widget accessible from the bottom panel</summary>
+  <url type="homepage">https://wiki.gnome.org/Apps/Gedit/ShippedPlugins</url>
+  <url type="bugtracker">https://gitlab.gnome.org/GNOME/gedit-plugins/issues</url>
```

Comment:
I suppose we have here a forked version, I think it makes sense to have mate url instead